exploreRiotAPI.code-workspace

```diff
@@ -3,5 +3,12 @@
 		{
 			"path": "."
 		}
-	]
+	],
+	"settings": {
```

Comment:
This commit changes a lot of quotes to apostrophes… is there a reason this section in particular is using quotes?

Reply:
Hmmm I thought in the official style guide, they recommended single quotes for names of object properties including key references in dicts and such. But after double checking it, it appears that's not the case. I may have read it on someone's blog or something lol. https://peps.python.org/pep-0008/#string-quotes

Reply:
Maybe you're thinking of CKAN
https://docs.ckan.org/en/ckan-2.1.5/python-coding-standards.html#:~:text=Use%20single%2Dquotes%20for%20string,ve%20got%20an%20error!%22%20.

```Use single-quotes for string literals, e.g. 'my-identifier', but use double-quotes for strings that are likely to contain single-quote characters as part of the string itself (such as error messages, or any strings containing natural language), e.g. "You've got an error!".

Single-quotes are easier to read and to type, but if a string contains single-quote characters then double-quotes are better than escaping the single-quote characters or wrapping the string in double single-quotes.```

I think this makes sense. If this is the rule you are following I think "settings" here would use single quotes since it's a dictionary key. (This is also pretty nitpicky of me)